staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/controller.go

```diff
@@ -358,13 +361,13 @@ func (c *celAdmissionController) Validate(
 			for _, decision := range decisions {
 				switch decision.kind {
 				case admit:
-					// TODO: add metrics for ignored error here
+					celmetrics.Metrics.ObserveCheck(ctx, decision.elapsed, definition.Name, binding.Name, decision.expression, "admit", definition.Spec.ParamKind.Kind, "active")
```

Comment:
We only want to use metric when there is an error occurred but ignored because of failure policy.

Reply:
We could watch for all admitted request depending on how the metrics defined. But will `definition.Spec.ParamKind.Kind` got nil pointer?

Reply:
Right, I'll mark this as resolved when the check is only observed under the `ignore` failure policy. And I think you're right about the nil pointer check; the metric observation should be unreachable if `ParamKind` is nil, since that's an invalid policy state, so I'll remove that too.

Reply:
(resolving since it seems the nil check can't be removed and the discussion on counting all admissions vs. counting only ones with errors is now over [here](https://github.com/kubernetes/kubernetes/pull/113475#discussion_r1016047160))